extensions/blocks/eventbrite/index.js

```diff
@@ -44,6 +44,7 @@ export const settings = {
 	keywords: [ __( 'events', 'jetpack' ), __( 'tickets', 'jetpack' ) ],
 	supports: {
 		html: false,
+		align: [ 'left', 'center', 'right' ],
```

Comment:
Any specific reason we can't support all alignment options such as wide and full? It may be useful in some themes.

Reply:
Not sure, but I also realised I think this line can be `align: true`.